### PR TITLE
fix(website): docs path redirect

### PR DESCRIPTION
**Pull request by aarongarciah**
The docs are a static site hosted at Netlify, so when accessing these urls 

- https://modulz-radix.netlify.com/docs
- https://modulz-radix.netlify.com/docs/ (note the trailing slash)

a 404 page is shown.

This PR solves that issue with the [gatsby-plugin-netlify](https://www.gatsbyjs.org/packages/gatsby-plugin-netlify/) that automatically generates a [`_redirects`](https://www.netlify.com/docs/redirects/) Netlify file with the redirects created from Gatsby with the `createRedirect` actions.

### CODE REVIEW

**Comment by aarongarciah** [comment]
Another approach not attached to Netlify is to use the [gatsby-plugin-client-side-redirect](https://www.gatsbyjs.org/packages/gatsby-plugin-client-side-redirect/) that generates `index.html` files with client side redirects automatically from the `createRedirect` action calls.

What do you think would be a better approach? I like the _gatsby-plugin-client-side-redirect_ one because is not tied to Netlify.

File: packages/website/gatsby-node.js [hunk 1]
```diff
@@ -49,14 +50,21 @@ exports.createPages = async ({ graphql, actions }) => {
...
   createRedirect({
     fromPath: `/docs`,
     isPermanent: true,
     redirectInBrowser: true,
-    toPath: `/docs/getting-started`,
+    toPath: defaultPath,
+  });
+
+  createRedirect({
+    fromPath: `/docs/`,
```
Comment (peduarte) [inline packages/website/gatsby-node.js]: So even with the netlify plugin, we need to create redirects from both `/docs` and `/docs/`?

**Comment by peduarte** [comment]
@aarongarciah Nice, I think your initial approach is fine for now 👌

File: packages/website/gatsby-node.js (hunk 1 above)
Comment (aarongarciah) [reply packages/website/gatsby-node.js]: It seems so. I will do another test later just to check again.

Comment (peduarte) [reply packages/website/gatsby-node.js]: Ok, I'll wait for your report 🤙

Comment (aarongarciah) [reply packages/website/gatsby-node.js]: Tested just now and the later redirect is not needed. [Netlify treats redirect rules](https://www.netlify.com/docs/redirects/#trailing-slash) always without trailing slash, so these two rules behave like the first one in the end:

`/docs` -> `/docs/getting-started`
`/docs/` -> `/docs/getting-started`

Comment (peduarte) [reply packages/website/gatsby-node.js]: Even better 🙂

**Comment by peduarte** [comment]
Will wait for the Netlify build so I can test and then 🚀

**Pull request merged**